rest/src/main/java/io/syndesis/rest/v1/handler/integration/IntegrationHandler.java

```diff
@@ -151,7 +151,6 @@ public void update(String id, @ConvertGroup(from = Default.class, to = AllValida
         Integration updatedIntegration = new Integration.Builder()
             .createFrom(integration)
             .deployedRevisionId(existing.getDeployedRevisionId())
-            .userId(Tokens.getUsername())
```

Comment:
Guess its ok to remove this hear as the user has been taken over from the original integration.